components/classes/location.json

```diff
@@ -0,0 +1,50 @@
+{
+  "meta:license": [
+    "Copyright 2022 Bounteous. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/location",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Live Event Location",
```

Comment:
Let's just call it as "Location" as it could be used for other conferences too or something other than a conventional event

Reply:
np, changed.